config/routes.rb

```diff
@@ -1,5 +1,13 @@
 Rails.application.routes.draw do
+  get 'puppies/index'
+  get 'puppies/show'
+  get 'puppies/new'
```

Comment:
Maybe we should delete those, since we use "resources"

Reply:
good catch! done